android_16/resources/products/product_reader.py

```diff
@@ -0,0 +1,14 @@
+import os
+from csv import reader
+
+from android_16.resources.products.product import Product
+
+
+class ProductReader:
+    CSV_NAME = f'{os.getcwd()}/android_16/data_store/products.csv'
```

Comment:
Seems a little hacky, but fine for now as we're eventually going to move from CSV to MySQL